addons/website_hr_recruitment/views/website_hr_recruitment_templates.xml

```diff
@@ -3,81 +3,126 @@
-                            <div class="card card-default mb32">
-                                <div class="card-body" t-att-data-publish="job.website_published and 'on' or 'off'">
-                                    <span t-if="not job.website_published" class="badge text-bg-danger mb8 p-2">unpublished</span>
-                                    <h3 class="text-secondary mt0 mb4">
-                                        <span t-field="job.name"/>
-                                    </h3>
-                                    <h5 t-if="job.no_of_recruitment &gt;= 1">
-                                        <t t-esc="job.no_of_recruitment"/> open positions
-                                    </h5>
-                                    <div t-if="editable"
-                                        t-field="job.description"
-                                        class="mt16 mb0 css_non_editable_mode_hidden"/>
-                                    <div t-esc="job.description or ''"
-                                        class="mt16 mb0 css_editable_mode_hidden o_website_hr_recruitment_job_description"
-                                    />
-                                    <div class="o_job_infos mt16">
-                                        <span t-field="job.address_id" t-options='{
-                                            "widget": "contact",
-                                            "fields": ["address"],
-                                            "no_tag_br": True
-                                            }'/>
-                                        <div t-if="job.contract_type_id">
-                                            <i class="fa fa-fw fa-suitcase" title="Employment type" role="img" aria-label="Employment type"/><span t-field="job.contract_type_id"/>
-                                        </div>
-                                        <div>
-                                            <i class="fa fa-fw fa-clock-o" title="Publication date" role="img" aria-label="Publication date"/><span t-field="job.write_date"/>
-                                        </div>
-                                    </div>
+                            <div class="alert alert-info p-3 mx-lg-4" groups="hr_recruitment.group_hr_recruitment_manager">
+                                <div class="text-center text-muted">
+                                    <h5>Create new job pages from the <strong>+ <i>New</i></strong> top-right button.</h5>
                                 </div>
                             </div>
-                        </a>
+                            <div class="col-lg-12 text-center pt24">
+                                <h5 class="fw-light text-muted">There are currently no open job opportunities,</h5>
+                                <h5 class="fw-light text-muted pb24">but feel free to <span class="fw-bold">contact us</span> for a spontaneous application.</h5>
```

Comment:
A sentence split over two h5... not only this is bad for a good html structure but this is awful for translations...
I guess you did not like the line-height of a h5 with a <br> inside... but that would be the right way to go :grimacing: I'll go with that but with an extra mb class on the <br> to have what you wanted. We should probably just review all headings line-height though... @stefanorigano 

I also change from h5 to p.h5, don't like those being title.

In the end, by chance:
```xml
<h5>a</h5>
<h5>b</h5>
```
is the pixel-exact-equivalent as
```xml
<h5>a<br class="mb-2"/>b</h5>
```
but the second is a valid structure and translation-friendly.